examples/client.rs

```diff
@@ -133,6 +124,23 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+async fn handle_request<T: quic::BidiStream<Bytes>>(
```

Comment:
This is just my thoughts as I read this, feel free to disagree with me. But I feel the example was easier to read without the extra function. At least as an initial first glance, "how do I do anything with the client". I'd consider showing something like this `handle_request` function in the documentation for the `client` module or something.

🤷

Reply:
You are right, it adds complexity to show something that belongs to the docs.